src/components/Followers/Followers.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from 'react-router-dom'
+import { useQuery, useMutation } from "@apollo/client";
+import { useState, useEffect } from "react";
+import FollowerDetails from '../FollowerDetails/FollowerDetails'
+import './Followers.css'
+
+import * as gql from '../../queries/queries'
+
+const Followers = ({id}) => {
+
+  const { loading, error, data } = useQuery(gql.GET_USER_INFO(id))
+  const [createFollower] = useMutation(gql.CREATE_FOLLOWER, {
```

Comment:
I think we went through this before, but I like the way you're using object notation on gql, to make sure you have access to all the different queries